src/core/mac/mac.cpp

```diff
@@ -873,5 +943,12 @@ Whitelist &Mac::GetWhitelist(void)
     return mWhitelist;
 }
 
+otMacCounters &Mac::GetCounters(void)
+{
+    mCounter.mTxTotal = mCounter.mTxData + mCounter.mTxDataPoll + mCounter.mTxBeacon + mCounter.mTxBeaconRequest;
+    mCounter.mRxTotal = mCounter.mRxData + mCounter.mRxDataPoll + mCounter.mRxBeacon + mCounter.mRxBeaconRequest;
```

Comment:
We should keep track of these totals separately, especially since we are not keeping track of all IEEE 802.15.4 frames that may be generated by non-Thread devices.